docs/get-started.md

```diff
@@ -0,0 +1,45 @@
+# Get started with `wksctl`
+
+Using `wksctl` you have two modes of operation. **Standalone** mode and **GitOps** mode. The latter will enable you to keep the state of the cluster itself in Git too.
+
+## Modes of use
+
+In **standalone mode**, `wksctl` builds a static cluster based on the contents of `cluster.yaml` and `machines.yaml` files passed on the command line; in **GitOps mode**, changes to `cluster.yaml` and `machines.yaml` files stored in Git will cause updates to the state of the live cluster.
+
+### Standalone mode
+
+Run `wksctl apply` and pass in the paths to `cluster.yaml` and `machines.yaml`
+
+```console
+wksctl apply \
+  --cluster cluster.yaml \
+  --machines machines.yaml
+```
+
+### GitOps mode
+
+We will create a cluster by pulling the cluster and machine yaml from git.
+
+The following are new commandline arguments to `wksctl apply` which will result in a cluster being created.
+
+- **git-url** The git repo url containing the `cluster.yaml` and `machine.yaml`
+- **git-branch**  The branch within the repo to pull the cluster info from
+- **git-deploy-key** The deploy key configured for the GitHub repo
+- **git-path** Relative path to files in Git (optional)
+
+The git command line arguments will be passed instead of `--cluster` and `--machines`.
+
+```console
+wksctl apply \
+  --git-url git@github.com:meseeks/config-repo.git \
```

Comment:
I see this was already here.  although I love the rick and morty reference, github.com/meseeks is a repo owned by someone else outside weaveworks.  We have been trying to avoid such references because (1) this is an attack vector if this person decides to make a repo named `config-repo`, (2) this is an area that we've had a security bounty called in for this kind of thing in the past (suggesting that people are going around looking for this kind of attack).

```suggestion
  --git-url git@github.com:$YOUR_GITHUB_ORG/config-repo.git \
```

or maybe something like
```suggestion
  --git-url git@github.com:$GITHUB_ORG/$CONFIG_REPO.git \
```
since the `config-repo` is basically mocked in there anyway as well (i.e. we don't expect or require the repo to be named `config-repo`).

Reply:
Yes - well spotted!